src/internal/m365/exchange/transform.go

```diff
@@ -210,26 +210,26 @@ func toItemAttachment(orig models.Attachmentable) (models.Attachmentable, error)
 }
 
 // TODO #2428 (dadam39): re-apply nested attachments for itemAttachments
-// func sanitizeAttachments(attached []models.Attachmentable) ([]models.Attachmentable, error) {
-// 	attachments := make([]models.Attachmentable, len(attached))
+func sanitizeAttachments(attached []models.Attachmentable) ([]models.Attachmentable, error) {
+	attachments := make([]models.Attachmentable, len(attached))
```

Comment:
You've got the classic [non-zero slice gotcha](https://go.dev/play/p/9IFsvXU6cNm) here.